packages/app/index.html

```diff
@@ -3,9 +3,9 @@
 
 <head>
   <meta charset="utf-8" />
-  <link href="/__cypress/assets/favicon.png" rel="icon">
+  <link href="/__cypress/assets/favicon.png?2022" rel="icon">
```

Comment:
is this change necessary? i feel like this could use a comment explaining why there is a number here